dvc.yaml

```diff
@@ -0,0 +1,27 @@
+stages:
+  prepare:
```

Comment:
isn't this done in #23 as well? There are going to be some conflicts

Reply:
#23 branched off of the branch in this PR